modules/msp/resource/utils/common.go

```diff
@@ -87,3 +88,18 @@ func AesDecrypt(data string) string {
 
 	return string(plainText)
 }
+
+// SetlabelsFromOptions set service labels from addon Options
+func SetlabelsFromOptions(options, labels map[string]string) {
+	// addons[xxx].options 中指定的 Option 创建为 labels
+	for k, v := range options {
```

Comment:
Just a map, no range

Reply:
Need to determine whether to fill in AddonVolumeSize when filling in AddonDiskType

Reply:
resolved